connect-examples/v2/java_payment/src/main/java/com/squareup/connectexamples/ecommerce/Main.java

```diff
@@ -99,21 +99,13 @@ String charge(@ModelAttribute NonceForm form, Map<String, Object> model) throws
         TransactionsApi transactionsApi = new TransactionsApi();
         transactionsApi.setApiClient(squareClient);
 
-        ChargeResponse response = transactionsApi.charge(squareLocation.getId(), chargeRequest);
+        ChargeResponse response = transactionsApi.charge(squareLocationId, chargeRequest);
 
         model.put("transactionId", response.getTransaction().getId());
 
-        return "charge";
-    }
-
-    private Location lookupCardProcessingLocation() throws ApiException {
-        LocationsApi locationsApi = new LocationsApi();
-        locationsApi.setApiClient(squareClient);
+        // Print the object to the console just to see it
+        System.out.print(response.getTransaction());
```

Comment:
nit: Is this print statement needed? In general, this should go to a log instead of console, but this is just an example.